DEBUG 2021-04-21 15:16:12.389 UTC: 	 CRAB Client version: v3.210323
DEBUG 2021-04-21 15:16:12.389 UTC: 	 Running on: Linux cmslpc173.fnal.gov 3.10.0-1160.6.1.el7.x86_64 #1 SMP Tue Nov 10 08:19:23 CST 2020 x86_64 x86_64 x86_64 GNU/Linux - Scientific Linux 7.9 (Nitrogen)
DEBUG 2021-04-21 15:16:12.389 UTC: 	 Executing command: 'status'
DEBUG 2021-04-21 15:16:12.390 UTC: 	 Found CRAB cache file /uscms/home/cannaert/.crab3
ERROR 2021-04-21 15:16:12.391 UTC: 	 Error: crab_projects/crab_MC_prod_chi850GeV_Suu4TeV_MINIAOD_000 is not a valid CRAB project directory.
ERROR 2021-04-21 15:16:12.391 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/status.py", line 39, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/SubCommand.py", line 639, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_projects/crab_MC_prod_chi850GeV_Suu4TeV_MINIAOD_000 is not a valid CRAB project directory.
INFO 2021-04-21 15:16:12.394 UTC: 	 Log file is /uscms_data/d3/cannaert/UL2018_genproduction/Suu4TeV_chi850GeV/CMSSW_10_6_20/src/crab.log
DEBUG 2021-04-22 10:33:02.303 UTC: 	 CRAB Client version: v3.210323
DEBUG 2021-04-22 10:33:02.304 UTC: 	 Running on: Linux cmslpc168.fnal.gov 3.10.0-1160.21.1.el7.x86_64 #1 SMP Tue Mar 16 08:00:44 CDT 2021 x86_64 x86_64 x86_64 GNU/Linux - Scientific Linux 7.9 (Nitrogen)
DEBUG 2021-04-22 10:33:02.304 UTC: 	 Executing command: 'status'
DEBUG 2021-04-22 10:33:02.304 UTC: 	 Found CRAB cache file /uscms/home/cannaert/.crab3
ERROR 2021-04-22 10:33:02.308 UTC: 	 Error: crab_projects/crab_MC_prod_chi850GeV_Suu4TeV_MINIAOD_000 is not a valid CRAB project directory.
ERROR 2021-04-22 10:33:02.308 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/status.py", line 39, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/SubCommand.py", line 639, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_projects/crab_MC_prod_chi850GeV_Suu4TeV_MINIAOD_000 is not a valid CRAB project directory.
INFO 2021-04-22 10:33:02.311 UTC: 	 Log file is /uscms_data/d3/cannaert/UL2018_genproduction/Suu4TeV_chi850GeV/CMSSW_10_6_20/src/crab.log
DEBUG 2021-04-22 16:00:56.556 UTC: 	 CRAB Client version: v3.210323
DEBUG 2021-04-22 16:00:56.557 UTC: 	 Running on: Linux cmslpc119.fnal.gov 3.10.0-1160.21.1.el7.x86_64 #1 SMP Tue Mar 16 08:00:44 CDT 2021 x86_64 x86_64 x86_64 GNU/Linux - Scientific Linux 7.9 (Nitrogen)
DEBUG 2021-04-22 16:00:56.557 UTC: 	 Executing command: 'status'
DEBUG 2021-04-22 16:00:56.557 UTC: 	 Found CRAB cache file /uscms/home/cannaert/.crab3
ERROR 2021-04-22 16:00:56.565 UTC: 	 Error: crab_projects/crab_MC_prod_chi850GeV_Suu4TeV_MINIAOD_000 is not a valid CRAB project directory.
ERROR 2021-04-22 16:00:56.565 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/status.py", line 39, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210323.00/lib/CRABClient/Commands/SubCommand.py", line 639, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_projects/crab_MC_prod_chi850GeV_Suu4TeV_MINIAOD_000 is not a valid CRAB project directory.
INFO 2021-04-22 16:00:56.569 UTC: 	 Log file is /uscms_data/d3/cannaert/UL2018_genproduction/Suu4TeV_chi850GeV/CMSSW_10_6_20/src/crab.log
